napari_allencell_annotator/_tests/utils/test_image_utils.py

```diff
@@ -0,0 +1,49 @@
+from pathlib import Path
+import numpy as np
+import pytest
+
+from napari_allencell_annotator.util.image_utils import ImageUtils
+
+
+def test_get_dask_data_tiff() -> None:
+    # ACT
+    test_image = ImageUtils(
+        Path("napari_allencell_annotator/_tests/assets/image_types/test_img.ome.tiff")
+    ).get_dask_data()
+
+    # ASSERT
+    assert np.all(test_image == np.array([[[0, 0, 0], [0, 0, 0]], [[0, 0, 0], [0, 0, 0]]]))
```

Comment:
whats the benefit of using `all` over `array_equal`? I think `array_equal` is a safer bet to check the read image

Reply:
`np.array_equal()` returns `False`. I think it's because of the precision problem in representing floating point values, making each element in test_image not exactly equal to 0. So I'm using `np.allclose()` instead

Reply:
in https://github.com/aics-int/napari-allencell-annotator/pull/77/commits/e73e1247ce5a1c668b5dee16970f559d4c1217c4